pkg/deploy/controller/deployment/controller.go

```diff
@@ -158,15 +163,14 @@ func (c *DeploymentController) Handle(deployment *kapi.ReplicationController) er
 			// If the deployment is cancelled here then we deleted the deployer in a previous
 			// resync of the deployment.
 			if !deployutil.IsDeploymentCancelled(deployment) {
-				deployment.Annotations[deployapi.DeploymentStatusAnnotation] = string(nextStatus)
-				deployment.Annotations[deployapi.DeploymentStatusReasonAnnotation] = deployapi.DeploymentFailedDeployerPodNoLongerExists
+				updatedAnnotations[deployapi.DeploymentStatusReasonAnnotation] = deployapi.DeploymentFailedDeployerPodNoLongerExists
 				c.emitDeploymentEvent(deployment, kapi.EventTypeWarning, "Failed", fmt.Sprintf("Deployer pod %q has gone missing", deployerPodName))
-				glog.V(4).Infof("Failing deployment %q because its deployer pod %q disappeared", deployutil.LabelForDeployment(deployment), deployerPodName)
+				glog.V(2).Infof("Failing deployment %q because its deployer pod %q disappeared", deployutil.LabelForDeployment(deployment), deployerPodName)
```

Comment:
Controller failures should be v4 if they will happen in normal use.  V2 is
for "we did not expect this to happen".

On Jul 21, 2016, at 2:45 AM, Michail Kargakis <notifications@github.com>
wrote:

In pkg/deploy/controller/deployment/controller.go
<https://github.com/openshift/origin/pull/9970#discussion_r71654205>:

>  				c.emitDeploymentEvent(deployment, kapi.EventTypeWarning, "Failed", fmt.Sprintf("Deployer pod %q has gone missing", deployerPodName))
> -				glog.V(4).Infof("Failing deployment %q because its deployer pod %q disappeared", deployutil.LabelForDeployment(deployment), deployerPodName)
> +				glog.V(2).Infof("Failing deployment %q because its deployer pod %q disappeared", deployutil.LabelForDeployment(deployment), deployerPodName)

INFO: >=4
WARN: >=2
ERR: >= 0

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9970/files/243f2894ba329617dab910eacd4c935f5ba9d49f#r71654205>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABG_p0eY9tv2cBnDIyLxHIPrkKFej3gAks5qXxWggaJpZM4JRTaV>
.